scripts/extensions/sams/src/components/assets/assetFilterPanel.tsx

```diff
@@ -79,6 +87,54 @@ export class AssetFilterPanel extends React.PureComponent<IProps, IState> {
         };
     }
 
+    setLocalAssetTagSearchParams(tag: IAssetTag, method?: string) {
+        if (method === 'add') {
+            this.addTag(tag);
+        } else if (method === 'remove') {
+            this.removeTag(tag);
+        }
+    }
+
+    addTag(value: IAssetTag) {
+        this.setState((preState) => {
+            const oldState = preState;
+            const tags: Array<IAssetTag> = preState.localSearchParams.tags! || [];
+            const index = tags.findIndex((tag) => {
+                return tag.code === value.code;
+            });
+
+            if (index === -1) {
+                tags!.push(value);
+                return {
+                    localSearchParams: {
```

Comment:
Does this override any other search params currently set?

Reply:
No. It only updates the existing tags with new ones appended to them.